### PR TITLE
Refactor SDKs and Simulators

### PR DESCRIPTION
**Pull request by ROSeaboyer**

### CODE REVIEW

File: osFiles/Simulators/iOS/20A360.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "osStr": "iOS",
+    "version": "16 Simulator Runtime",
```
Comment (dhinakg) [inline osFiles/Simulators/iOS/20A360.json:3]: inconsistent naming

Comment (ROSeaboyer) [reply osFiles/Simulators/iOS/20A360.json:3]: which way makes more sense to you? I'm leaning towards "Simulator" but can go either way

Comment (dhinakg) [reply osFiles/Simulators/iOS/20A360.json]: agreed

Comment (ROSeaboyer) [reply osFiles/Simulators/iOS/20A360.json]: Fixed and pushed

**Pull request merged**